test/controllers/graphql_controller_v2_test.rb

```diff
@@ -408,8 +402,40 @@ def setup
       assert_equal(1, filtered_source_array.count)
 
       expected_sources = ['dspace@mit']
-      actual_sources = filtered_source_array.map{|source| source["key"]}
+      actual_sources = filtered_source_array.map { |source| source['key'] }
       assert_equal(expected_sources, actual_sources)
     end
   end
+
+  test 'graphqlv2 can retrieve a record from a default index' do
```

Comment:
Ok - I don't think we should remove this test to be clear. Reading through the file, I see this test as paired with the adjacent one that changes the index value, while the earlier test is removed enough that it isn't clear that we're testing both conditions without a higher cognitive load. While this may be a small amount of over-testing, I don't think that's particularly an issue. The suite runs pretty quickly, so there's no downside IMO.